term/theme.json

```diff
@@ -36,7 +134,10 @@
 			}
 		},
 		"typography": {
-			"fontFamily": "var(--wp--preset--font-family--system-font)"
+			"fontFamily": "var(--wp--preset--font-family--system-font)",
+			"fontSize": "var(--wp--preset--font-size--medium)",
+			"lineHeight": "1.5",
+			"textTransform": "capitalize"
```

Comment:
I think the body font weight needs to be thinner (but maybe it's my mac, that doesn't have helvetica!)

Reply:
On mine, I can see the font in a size size, but I can definitely change it back as it was, or not set a default. What do you think is the best way here ?